packages/lesswrong/components/tagging/FilterMode.tsx

```diff
@@ -272,7 +284,10 @@ const FilterModeRawComponent = ({tagId="", label, mode, canRemove=false, onChang
             {description}
           </ContentStyles>}
         </div>
-        {tag && <TagPreview tag={tag} showCount={false} postCount={3}/>}
+        {tag &&
+          <div className={classes.tagPreview}>
+            <TagPreview tag={tag} showCount={false} postCount={tagPreviewPostCount}/>
```

Comment:
@jpaddison3 this would hide all links to posts under a tag - currently `tagPreviewPostCount` is set to 0 for LW via a forumSelect earlier, but keeps the old behavior for EA forum.  Do you also want the tag preview to no longer show 3 posts as a preview of posts for that tag?

Reply:
Correct.